### ISSUE TITLE
Atypical Teratoid/Rhabdoid Tumor

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7747]
**Preferred term label:**
Atypical Teratoid/Rhabdoid Tumor

**Synonyms**

**Definition (free text, please give PubMed ID)**

An aggressive malignant embryonal neoplasm arising from the central nervous system. It is composed of cells with a large eccentric nucleus, prominent nucleolus, and abundant cytoplasm. Mutations of the SMARCB1 gene or very rarely SMARCA4 (BRG1) gene are present. The vast majority of cases occur in childhood. Symptoms include lethargy, vomiting, cranial nerve palsy, headache, and hemiplegia.

also Rhabdoid Tumor of the Kidney
(NCIT:C8715) A rhabdoid tumor that arises from the kidney. It occurs in children and it is associated with abnormalities of chromosome 22. It is characterized by the presence of cells with a large eccentric nucleus, prominent nucleolus, and abundant cytoplasm. 

**Parent term (use hpo.jax.org/app)**
Embryonal neoplasm HP:0002898

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
613325 
PMID: 10739764  1/2
also 1/2 for RTK

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
